elements/lisk-p2p/src/p2p.ts

```diff
@@ -27,7 +27,7 @@ interface SCServerUpdated extends SCServer {
 import {
 	constructPeerId,
 	constructPeerIdFromPeerInfo,
-	REMOTE_RPC_GET_ALL_PEERS_LIST,
+	REMOTE_RPC_GET_MINIMAL_PEERS_LIST,
```

Comment:
This variable name seems to indicate the list is minimal i.e. reduced. Maybe rename to `REMOTE_RPC_GET_BASIC_PEERS_LIST ` ?